py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -471,6 +471,9 @@ def _request(self, method, url, body=None):
             request.add_header('Accept', 'application/json')
             request.add_header('Content-Type', 'application/json;charset=UTF-8')
 
+            base64string = base64.b64encode('%s:%s' % (parsed_url.username, parsed_url.password))
```

Comment:
I don't know. It seems like it should go in one of the else blocks one or two levels above. Probably in the request object that doesn't have a password manager.